### ISSUE TITLE
Blue warning icon has insufficient contrast

### ISSUE THREAD

**Issue by Floppy** [manyfold3d/manyfold#2099]
The blue warning icon does not have enough contrast against the background (this should be at least 3:1).

See https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html

**Comment by Floppy** [comment]
I can't verify or replicate this, it might be better now after some other changes. The only contrast problem I can see if the blue links against the table background in the problem list.
